src/locales/pl/PAGES/SCOPE.json

```diff
@@ -15,11 +15,11 @@
     "WCAG21": "WCAG 2.1",
     "WCAG20": "WCAG 2.0",
     "LABEL_CONFORMANCE_TGT": "Poziom zgodności",
-    "INF_CONF_TGT": "Wybierz docelowy poziom zgodności z WCAG 2 («A», «AA» lub «AAA») do oceny. Aby uzyskać więcej informacji, zobacz [WCAG-EM Krok 1.b: Określ wymagany poziom zgodności](https://testy.lepszyweb.pl/met-wcag-em-pl#step1b). Ten wybór decyduje, które filtry poziomu zgodności będą domyślnie aktywne w „kroku 4: Audyt próbki”.",
+    "INF_CONF_TGT": "Wybierz docelowy poziom zgodności z WCAG 2 («A», «AA» lub «AAA») do oceny. Aby uzyskać więcej informacji, zobacz [WCAG-EM Krok 1.b: Określ wymagany poziom zgodności](https://www.w3.org/TR/WCAG-EM/#step1b). Ten wybór decyduje, które filtry poziomu zgodności będą domyślnie aktywne w „kroku 4: Audyt próbki”.",
     "LABEL_SUPPORT_BASE": "Podstawowy poziom obsługi dostępności",
-    "INF_SUPPORT_BASE": "Określ minimalny zestaw kombinacji przeglądarek, technologii wspomagających i innych [programów użytkownika](https://www.w3.org/Translations/WCAG21-pl/#dfn-programy-uzytkownika), w których witryna ma być [obsługiwana pod względem dostępności](https://www.w3.org/Translations/WCAG21-pl/#dfn-obs-ugujaca-dostepnosc).  Na przykład, „Firefox z NVDA”, „Internet Explorer (IE) z JAWS” i „Apple z VoiceOver” mogą być takim podstawowym zestawem. Więcej informacji można znaleźć w części [WCAG-EM Krok 1.c: Określ poziom obsługi dostępności](https://testy.lepszyweb.pl/met-wcag-em-pl#step1c).",
```

Comment:
@zwiastunsw It is ok to link to the WCAG 2.1 Authorized Translation into Polish. [More info in the translation notes.](https://github.com/w3c/wai-wcag-em-report-tool/wiki/How-to-add-a-language#add-wcag-translations)